content/800-data-platform/050-data-proxy/700-prisma-cli-with-data-proxy.mdx

```diff
@@ -9,19 +9,9 @@ tocDepth: 3
 
 Currently, you can only use [Prisma Client](/concepts/components/prisma-client) with the Data Proxy to _query_ your database.
 
-</TopBlock>
-
+You need a direct (non-Data Proxy) database connection if you apply schema changes to your database with [Prisma Migrate](/concepts/components/prisma-migrate/get-started) or [`db push`](/concepts/components/prisma-migrate/db-push), or if you [introspect](/concepts/components/introspection) your database.
 
-### Problem
-If you want to run [migrations](/concepts/components/prisma-migrate/get-started), [introspection](/concepts/components/introspection),  or any of the [Prisma CLI commands](/reference/api-reference/command-reference) listed below, you can only do so on a direct (non-Data Proxy) database connection.
-
-<Admonition type="warning">
-
-**Important**<br /><br />
-
-If your database is behind a firewall, add your local IP address to the allowlist of your database.
-
-</Admonition>
+The following [Prisma CLI commands](/reference/api-reference/command-reference) require a direct connection:
```

Comment:
```suggestion

## Prisma CLI commands that require a direct database connection

The following [Prisma CLI commands](/reference/api-reference/command-reference) require a direct connection:
```

Reply:
Adding a second heading which separates the introduction and acts as a reference for the CLI commands that are impacted. Also, avoids having a single heading in the On this page TOC.

Reply:
Agree this is better. I had to move some other things around too, so did it as a separate commit.